gluon-mesh-vpn-wireguard-vxlan/Makefile

```diff
@@ -0,0 +1,36 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-mesh-vpn-wireguard
+PKG_VERSION:=1
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/gluon-mesh-vpn-wireguard
+  SECTION:=gluon
+  CATEGORY:=Gluon
+  TITLE:=Support for connecting meshes via wireguard
+  DEPENDS:=+gluon-mesh-vpn-core +gluon-config-mode-core +micrond +kmod-wireguard +wireguard-tools +curl +ca-bundle +ip-full
```

Comment:
For what is ip-full needed?

Also, `uclient` might be sufficient and I'm not sure whether or not i miss the attack vendor for submitting the private key.

Reply:
Which private-key are we talking about? ip-full was in the original package but you could be correct that uclient is enough.

Reply:
The private-key will obviously never leave the node.

Reply:
Yeah, i mean the public key.

`uclient` might be able to replace curl, as submitting post-data works with it too AFAIR. This is not connected to `ip-full`, i was wondering which commands the default `ip` implementation of OpenWrt lacks.

Reply:
> i was wondering which commands the default ip implementation of OpenWrt lacks.

It's not needed, I checked.